commands/user_e2e_test.go

```diff
@@ -136,3 +136,43 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestUpdateUserEmailCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("admin and local user can change user email", func(c client.Client) {
+		printer.Clean()
+		oldEmail := s.th.BasicUser2.Email
+		newEmail := "basicuser2@fakedomain.com"
+		err := updateUserEmailCmdF(c, &cobra.Command{}, []string{s.th.BasicUser2.Email, newEmail})
+		s.Require().Nil(err)
+
+		u, err := s.th.App.GetUser(s.th.BasicUser2.Id)
+		s.Require().Nil(err)
+		s.Require().Equal(newEmail, u.Email)
+
+		u.Email = oldEmail
+		_, err = s.th.App.UpdateUser(u, false)
+		s.Require().Nil(err)
+	})
+
+	s.Run("normal user doesn't have permission to change another user's email", func() {
+		printer.Clean()
+		newEmail := "basicuser2-change@fakedomain.com"
+		err := updateUserEmailCmdF(s.th.Client, &cobra.Command{}, []string{s.th.BasicUser2.Id, newEmail})
+		s.Require().EqualError(err, ": You do not have the appropriate permissions., ")
+
+		u, err := s.th.App.GetUser(s.th.BasicUser2.Id)
+		s.Require().Nil(err)
+		s.Require().NotEqual(newEmail, u.Email)
```

Comment:
nit: this check is redundant to the next one. It doesn't do any harm, but I'd say that only the latter is needed